metrics/network/remote_network/remote-networking-iperf3.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This will test bandwidth using iperf3
+# using a remote setup (host --> client container) and
+# (host --> server container)
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/remote-networking-test-common.sh"
+
+# Time to run the server using iperf3
+server_time=30
+
+# This function will measure the bandwidth using iperf3
+# using a container as a client running in one host with
+# another container as a server running in another host
+function iperf3_bandwidth {
```

Comment:
I wonder if this should be a function, I am not sure if another script will consume it